README.md

```diff
@@ -1,7 +1,45 @@
 [![Build Status](https://travis-ci.org/dedis/onet.svg?branch=master)](https://travis-ci.org/dedis/onet)
-[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg?branch=travis_24)](https://coveralls.io/github/dedis/onet?branch=travis_24)
+[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg)](https://coveralls.io/github/dedis/onet)
 
-# onet
+# The Cothority Network Library - CoNet
 
-Onet is a network library that is used to build distributed applications on a
-large scale basis.
+The Cothority-network (CoNet) is a library for simulation and deployment of
+decentralized, distributed protocols. It offers an abstraction for tree-based
+communications between thousands of nodes and is used both in research for 
+testing out new protocols and running simulations, as well as in production to
+deploy those protocols as a service in a distributed manner. A third part of the
+library is used to write applications that contact the services on remote servers.
+
+CoNet is developed by [DEDIS/EFPL](http://dedis.epfl.ch) as part of the
+[Cothority](https://github.com/dedis/cothority) project that aims to deploy
+a large number of conodes for distributed signing and related projects.
+
+## Documentation
+
+- To run and use a conode, have a look at 
+	[Cothority Node](https://github.com/cothority/conode/wiki)
```

Comment:
This should point at to the README no ? Since there should be a "quick usage" section or stg no?